package.json

```diff
@@ -4,17 +4,44 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "sass --watch src/scss:dist/css & live-server",
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "lint": "eslint .",
+    "format": "prettier --write",
+    "lint:cypress": "eslint cypress --config .eslintrc.cypress.json",
+    "start:server": "node server.js",
+    "test": "jest",
+    "test:e2e": "npm run start:server && npx cypress run",
+    "testMatch": [
+      "<rootDir>/unit-tests/**/*.test.js"
```

Comment:
I'm not sure what this is for

Reply:
Sorry let me update the branch again   I can send you a message on Discord  ‏I don’t want to interrupt if it's okay.